### ISSUE TITLE
Issue with umlauts when using the keywords call

### ISSUE THREAD

**Issue by ForsakenNGS** [manticoresoftware/manticoresearch-php#59]
Hi,

I'm having some trouble with the keywords regarding german umlauts. If I issue the call with:
```php
$keywords = $index->keywords(implode("* ", $query)."*", []);
```
Which results in this request being sent to the http-interface:
```php
    array (size=3)
      'url' => string 'http://127.0.0.1:9308/sql' (length=25)
      'headers' => 
        array (size=1)
          0 => string 'Content-Type: ' (length=14)
      'body' => string 'mode=raw&query=CALL KEYWORDS('k\xc3\xbcche*','ebiz_trader_ad_master_de')' (length=72)
```
And this result:

```php
    array (size=5)
      'columns' => 
        array (size=3)
          0 => 
            array (size=1)
              'qpos' => 
                array (size=1)
                  'type' => string 'string' (length=6)
          1 => 
            array (size=1)
              'tokenized' => 
                array (size=1)
                  'type' => string 'string' (length=6)
          2 => 
            array (size=1)
              'normalized' => 
                array (size=1)
                  'type' => string 'string' (length=6)
      'data' => 
        array (size=1)
          0 => 
            array (size=3)
              'qpos' => string '1' (length=1)
              'tokenized' => string 'kxc3xbcche*' (length=11)
              'normalized' => string 'kxc3xbcche*' (length=11)
      'total' => int 0
      'error' => string '' (length=0)
      'warning' => string '' (length=0)
```

Currently I went to using a raw sql query like follwing to work around it, which works, but is far from ideal:
```php
        $keywords = $this->manticoreClient()->sql([
            'mode' => 'raw',
            'body' => [
                "query" => "CALL KEYWORDS('".mysql_real_escape_string(implode("* ", $query)."*")."', '".mysql_real_escape_string($indexNameFull)."');"
            ]
        ]);
```

**Comment by adriannuta** [comment]
The problem is the  [Utils:;escape](https://github.com/manticoresoftware/manticoresearch-php/blob/a0937b6c2d31e06385365f618532f028fc7d6c1f/src/Manticoresearch/Utils.php#L8) function.  I don't remember why I picked that solution, but it doesn't work with UTF characters.

**Comment by sanikolaev** [comment]
@ForsakenNGS can you verify that it's now fixed in https://packagist.org/packages/manticoresoftware/manticoresearch-php#dev-master ?

**Comment by mikeburns** [comment]
bumping as I encountered this problem and also tracked it down to the Utils::escape function.  It would seem (to me at least) that it is indeed fixed in dev-master.  So... perhaps it could be merged into the main branch and marked as closed?

**Comment by sanikolaev** [comment]
> So... perhaps it could be merged into the main branch and marked as closed?

OK. Done. Included into release 1.7.0
